### PR TITLE
initial commit bogdan

### PR DESCRIPTION
**Pull request by SturmkitX**
Merge my mods.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# modifications
```
Comment (alexandrugrigor) [inline README.md]: please remove this part :)

**Review by alexandrugrigor — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SturmkitX) [reply README.md]: Done :)

File: main/main.go [hunk 2]
```diff
@@ -4,8 +4,10 @@ import "fmt"
 
 func main() {
 	fmt.Println("hello world!")
+	fmt.Println("Razvan Farte was here")
```
Comment (alexandrugrigor) [inline main/main.go:7]: how did this ended up here? 😄

Comment (SturmkitX) [reply main/main.go:7]: don't know, it was already on master :D

Comment (alexandrugrigor) [reply main/main.go:7]: well, that's why rebase > merge when you want to get your latest changes 😆

**Review by alexandrugrigor — APPROVED** [review]

**Pull request merged**